cargo-dist/src/cli.rs

```diff
@@ -133,18 +132,20 @@ pub enum Commands {
     #[clap(disable_version_flag = true)]
     #[clap(hide = true)]
     HelpMarkdown(HelpMarkdownArgs),
-}
-
-#[derive(Args)]
-pub struct ReleaseNotesArgs {
-    /// Get release notes for a specific version.
+    /// Get a quick summary of the status of your project
     ///
-    /// Otherwise the app's current version will be used.
-    #[clap(long)]
-    pub version: Version,
+    /// If you want to know what running your cargo-dist CI will produce,
+    /// this is the command for you! It should run the exact same logic and do some
+    /// basic integrity checks.
+    ///
+    /// This is roughly an alias for:
+    ///
+    ///     cargo dist manifest --artifacts=all --no-local-paths
+    #[clap(disable_version_flag = true)]
+    Status(StatusArgs),
```

Comment:
i definitely would prefer some more akin to plan as i think status is confusing (since it's telling you what *will* happen in the future, not what is currently happening)

Reply:
hmmm

Reply:
gonna merge this for now and we can rename before release if we want